desktop-widgets/mainwindow.h

```diff
@@ -88,6 +88,7 @@ class MainWindow : public QMainWindow {
 	ProfileWidget2 *graphics;
 	DivePlannerWidget *divePlannerWidget;
 	DiveListView *diveList;
+    FilterWidget2 *filterWidget2;
```

Comment:
whitespace

Reply:
Is this actually used outside of `MainWindow`'s constructor? If not, there seems to be no point to introduce another member-variable.

Reply:
hey, still whitespace... and what about @bstoeger 's comment?